### ISSUE TITLE
"Is next sentence" pre-training task availability for Language Modeling scripts

### ISSUE THREAD

**Issue by shabie** [huggingface/transformers#11458]
# 🚀 Feature request

BERT was trained on 2 pretraining tasks. The scripts [here ](https://github.com/huggingface/transformers/tree/master/examples/pytorch/language-modeling) in the repo for language modeling only provide scripts for masked language modeling.

Is there any specific reason for this?

I was hoping if those scripts could be extended to include "next sentence" pretraining task to remain faithful to the pretraining methodology used by BERT if I choose to further pretrain on some corpus.

## Motivation

BERT uses 2 pre-training tasks and the scripts provide only of them.

## Your contribution

I am not sure if I can extend the script myself. I will gladly look into them if I know the there aren't any good reasons for them not being provided by HF in the first place.

Thanks a lot!

**Comment by LysandreJik** [comment]
Related issues: https://github.com/huggingface/transformers/issues/1622, https://github.com/huggingface/transformers/issues/2898 and https://github.com/huggingface/transformers/issues/2166

However, example scripts are made to be very understandable and very easy to tweak, so modifying them to include the next sentence prediction objective for BERT shouldn't be complicated!

**Comment by rishabhstha** [comment]
Same issue? Anybody done it before?
